snap/snapcraft.yaml

```diff
@@ -18,6 +18,12 @@ apps:
       - network-observe
       - network
       - home
+  discover:
+    command: bin/discover-tempest-config
```

Comment:
the solution i found for that at the time was to use command-chains. example:
```
apps:
  discover-tempest-config:
    command-chain:
      - bin/set-ssl-envvars
    command: bin/discover-tempest-config
    plugs:
      - network
      - home

parts:
  tempestconf:
  ...
    override-prime: |
      craftctl default
      cat << EOF > bin/set-ssl-envvars
      #!/bin/bash
      export SSL_CERT_FILE="\${OS_CACERT}"
      exec "\${@}"
      EOF
      chmod +x bin/set-ssl-envvars
```
i didn't wanna create a new file, so this solution is creating the command-chain script dynamically during build time , but creating the script and then using the dump plugin is also possible. the idea stands though. The key point is, this script will execute before as the current user whenever the discover-tempest-binary is executed and the replace itself with the discover-tempest-plugin process.

Another solution could be using the snapcraft.yaml's `env` key. I am not sure if it is set dynamically for each invocation though.

Reply:
After discussion this with @mert-kirpici, I tested to usage of `SSL_CACERT_DIR` env variable. It was not working so I went with usage of environment in snapcraft.yaml instead of command-chain. It's working as we accepted.

Thanks @mert-kirpici